casper/src/main/resources/Registry.rho

```diff
@@ -247,6 +265,53 @@ in {
               }
             }
           }
+        } |
+        
+        contract TreeHashMapUpdater(@map, @nybList, @n, @len, update, @suffix, ret) = {
+          // Look up the value of the node at [map, nybList.slice(0, n + 1)
+          new valCh, restCh in {
+            nodeGet!((map, nybList.slice(0, n)), *valCh) |
+            for (@val <- valCh) {
+              if (n == len) {
+                // If we're at the end of the path, set the node to newVal.
```

Comment:
Comment is not related to _update_ but _set_.

Reply:
done